src/elements/ProjectSettingsGeneral/ProjectSettingsGeneral.js

```diff
@@ -101,7 +101,7 @@ const ProjectSettingsGeneral = ({
       setProjectIsInitialized(true)
 
       fetchProject(params.projectName)
-        .then(projectResponse => {
+        .then(({ data: projectResponse }) => {
```

Comment:
there is will be a bug with destructuring data if the response doesn't contain it